windows-installer/generate_sdl_agent_exe.ps1

```diff
@@ -175,7 +175,11 @@ $replacement = (Get-Content $replacement_file) -join("`r`n")
 ##############################
 #  STEP 5 - REMOVE UNNECESSARY FILES.
 ##############################
-
+# Below dll is required to put in staging_bindir to run C++ based extension
+# included gem such as winevt_c.
```

Comment:
Suggested comment phrasing:
```
# Save the C++ runtime DLL to allow running gems with C++ native extensions.
```

Reply:
I will modify the first part with your sentence but keep the rest which mentioned about winevt_c